test/test_streams.py

```diff
@@ -1,14 +0,0 @@
-import pendulum
-from tap_zendesk_chat.streams import break_into_intervals
-
-
-def test_intervals():
```

Comment:
just double checking, did we mean to remove this / we don't need this test anymore?

Reply:
These tests no longer run. They were written by the original authors of the software before we forked it.